src/test/java/org/betonquest/betonquest/objectives/EntityInteractObjectiveTest.java

```diff
@@ -1,12 +1,15 @@
 package org.betonquest.betonquest.objectives;
 
+import org.betonquest.betonquest.modules.logger.util.BetonQuestLoggerService;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * Test for {@link EntityInteractObjective} and its inner classes.
  */
+@ExtendWith(BetonQuestLoggerService.class)
```

Comment:
What is the reason for extending this test with the LoggerService?
```suggestion
```

Reply:
A Counting Objective actually creates a logger, so the tests fail without this